inspire/modules/formatter/views.py

```diff
@@ -36,3 +38,12 @@ def get_bibtex(recid):
     record = get_record(recid)
     bibtex = Bibtex(record).format()
     return jsonify({"result": bibtex, "recid": recid})
+
+
+@blueprint.route('/latex', methods=['GET', ])
+@wash_arguments({'recid': (int, 0),
+                 'latex_format': (text_type, "")})
+def get_latex(recid, latex_format):
+    record = get_record(recid)
```

Comment:
if no `recid` is passed and it defaults to `0`, does it crash here the call to `get_record`?